applications/content/src/docs/asciidoc/content_IT.adoc

```diff
@@ -0,0 +1,21 @@
+////
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+////
+= Panoramica sul gestore di contenuti
```

Comment:
I think this should be removed because it contains italian? text, no?

Reply:
The docbook file content exist with only one line
I have chosen to migrate it, and I will ask to the Italian people after, if they want to continue translation or remove it

Reply:
Ok, thanks Olivier!